llvm/test/tools/llvm-objcopy/ELF/add-invalid-note.test

```diff
@@ -0,0 +1,46 @@
+## Verify that --add-section warns on invalid note sections.
+
+## Add [namesz, descsz, type, name, desc] for a build id.
+
+## Notes should be padded on 8 bytes.
+# RUN: echo -e -n "\x04\x00\x00\x00" >  %t-miss-padding-note.bin
```

Comment:
`RUN: printf` is more popular to craft specific bytes.

Reply:
I wonder whether this should be changed to an assembler test. `REQUIRES: x86-registered-target` will be needed but the test will be easier to read.